Tests/MSFT_xWindowsOptionalFeature.Tests.ps1

```diff
@@ -0,0 +1,311 @@
+<# 
+.summary
+    Test suite for MSFT_xWindowsOptionalFeature.psm1
+#>
+[CmdletBinding()]
+param()
+
+Import-Module $PSScriptRoot\..\DSCResources\MSFT_xWindowsOptionalFeature\MSFT_xWindowsOptionalFeature.psm1
```

Comment:
Is that what we should be using for all tests now? I just copied a file from another resource in the module!